### PR TITLE
Add unlock external task functional

### PR DESCRIPTION
**Pull request by TheNord**
Hi!

This PR add unlock external task functional.

Before unlock:
image

After unlock:
image

### CODE REVIEW

File: src/components/detail-process-instance/ExternalTask.vue [hunk 1]
```diff
@@ -74,6 +75,22 @@ export default {
                     type: "error"
                 });
             })
+        },
+        unlockExtTask(extId) {
+            this.$api().post(`/external-task/${extId}/unlock`).then(response => {
+                this.$notify({
+                    group: "foo",
```
Comment (antonkomarev) [inline src/components/detail-process-instance/ExternalTask.vue:84]: foo? is it required?

File: src/components/detail-process-instance/ExternalTask.vue [hunk 2]
```diff
@@ -5,6 +5,7 @@
         <template slot="actions" slot-scope="{ row }">
             <b-btn size="sm" @click="retryExtTask(row.processInstanceId, row.id)">
                 <font-awesome-icon icon="redo" />Retry</b-btn>
+            <b-btn size="sm" class="ml-2" @click="unlockExtTask(row.id)">Unlock</b-btn>
```
Comment (antonkomarev) [inline src/components/detail-process-instance/ExternalTask.vue]: Retry ext task sending processInstanceId with task id. Is there a possibility to have 2 same ext task id in different process instances?

Comment (TheNord) [reply src/components/detail-process-instance/ExternalTask.vue]: This API accepts only the task id. The task id is unique for all process instances and it PKey in database.

File: src/components/detail-process-instance/ExternalTask.vue (hunk 1 above)
Comment (TheNord) [reply src/components/detail-process-instance/ExternalTask.vue:84]: Not required, but this group used in all another notifications
https://www.npmjs.com/package/vue-notification#groups

**Comment by antonkomarev** [comment]
Can we show unlock button only when task is locked?

File: src/components/detail-process-instance/ExternalTask.vue (hunk 1 above)
Comment (TheNord) [reply src/components/detail-process-instance/ExternalTask.vue:84]: Without a group, the notification is not displayed

**Comment by TheNord** [comment]
> Can we show unlock button only when task is locked?

Good suggestion, added a check, now the button is displayed only for blocked tasks

File: src/components/detail-process-instance/ExternalTask.vue (hunk 1 above)
Comment (KotskinKotskin) [reply src/components/detail-process-instance/ExternalTask.vue:84]: it`s placeholder for notifications

**Comment by KotskinKotskin** [comment]
hi, thanks for your contribution.

**Pull request merged**